src/components/Navbar.jsx

```diff
@@ -2,8 +2,38 @@ import React from "react";
 import { NavLink } from "react-router-dom";
 import { FaPaperPlane } from "react-icons/fa";
 
+function Navbar() {
+    return(
+        <NavWrapper className="navbar navbar-expand-sm navbar-light px-sm-5">  
+            <div className="ml-auto navbar-nav">
+                <Link to="/">
+                    <div className="nav-item text-dark px-4 font-weight-bold">
```

Comment:
There are two navbar component on this page! The one below is well written and scalable to delete this one on the top.